packages/aws-cdk/lib/cli/cli-config.ts

```diff
@@ -398,7 +398,7 @@ export async function makeConfig(): Promise<CliConfig> {
           'from-path': { type: 'string', desc: 'Path to a local custom template directory or multi-template repository', requiresArg: true, conflicts: ['lib-version'] },
           'template-path': { type: 'string', desc: 'Path to a specific template within a multi-template repository (requires --from-path)', requiresArg: true },
         },
-        check: '(argv: any) => { if (argv.templatePath && !argv.fromPath) { throw new Error("--template-path requires --from-path to be specified"); } return true; }',
+        implies: { 'template-path': 'from-path' },
```

Comment:
shouldn't this change break the existing test for init? Why doesn't it?

and the test needs to be updated as well

Reply:
I don't see a test for the `implies` logic anymore